src/main/java/knightminer/simplytea/data/gen/RecipeGenerator.java

```diff
@@ -108,9 +122,12 @@ protected void buildRecipes(Consumer<FinishedRecipe> consumer) {
 		addTeaWithBag(consumer, tea_leaf, teabag_green, cup_tea_green);
 		addTeaWithBag(consumer, black_tea, teabag_black, cup_tea_black);
 		addTeaWithBag(consumer, chorus_petal, teabag_chorus, cup_tea_chorus);
+		addTea(consumer, cup_water_hot, teapot_hot);
+		addTea(consumer, cup_frothed, teapot_frothed);
 
 		// advanced tea
 		addTea(consumer, cup_cocoa, Items.COCOA_BEANS, Items.COCOA_BEANS, teapot_frothed);
+		addTea(consumer, cup_cocoa, Items.COCOA_BEANS, cup_frothed);
 		addHoney(consumer, RecipeCategory.FOOD, cup_cocoa, tea_stick, CocoaItem.CINNAMON_TAG);
 		addTea(consumer, cup_tea_chai, teabag_black, tea_stick, teapot_frothed);
```

Comment:
Iced tea and chai both are not added using the helper above, so they need their hot water cup recipes here like you did with cocoa.

Reply:
It looks like iced tea doesn't even need a teapot at all. The recipe is just a cup, teabag, apple, and ice cubes.

Reply:
Huh, that doesn't seem right. I need to remember my reasoning for that. It might just be a bug as I could not use the helper with the tag.

I'll consider whether it should have a pot in the recipe after this PR is finished.

Reply:
I added the cup recipe for chai.

Another recipe that you could consider is frothed milk with cinnamon. I wasn't sure whether it made sense or not as hot water and frothed milk are supposed to be much less useful than the actual teas or cocoa, but it would be useful for frothed milk to also have the option to clear all status effects instead of just one.

Reply:
Frothed milk with cinnamon seems like something someone would drink. So sure, why not.